py/qqa/run.py

```diff
@@ -239,7 +239,7 @@ def run_qa(indir, outfile=None, qalist=None):
     qarunner = QARunner(qalist)
     return qarunner.run(indir, outfile=outfile)
 
-def make_plots(infile, outdir):
+def make_plots(infile, outdir, preprocdir=None, cameras=None, rawfile=None):
```

Comment:
Two more details: rawfile option is no longer needed, and please add docstring descriptions of `preprocdir` and `cameras` arguments.  Thanks.

Reply:
Ok I've added it; check out the 2 latest commits (bf04ddb and 96d49c4).